.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "LSP Test - Debug active test",
```

Comment:
Ohh. We should do this for CLI test too (can be in separate PR if you want).

Reply:
Absolutely, I'm now working on testing debug visualizer support for some data type but I can add some extra similar configs for other runners